k4EDM4hep2LcioConv/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ add_library(k4EDM4hep2LcioConv SHARED
   src/k4EDM4hep2LcioConv.cpp)
 
 target_include_directories(k4EDM4hep2LcioConv PUBLIC
+  ${LCIO_INCLUDE_DIRS}
```

Comment:
There are targets for LCIO, right? the target_link_libraries should be that target (`LCIO::LCIO`) instead of `${LCIO_LIBRARIES}`

Reply:
That'd be nice indeed, but I don't think there are? https://github.com/iLCSoft/LCIO/search?l=CMake&q=Target

Reply:
Yeah, something I do in DD4hep :cry: https://github.com/AIDASoft/DD4hep/blob/33a586b25bb297a511bb0b99d0d6e7c34bb87056/cmake/DD4hepBuild.cmake#L899